debugging/book-library/script.js

```diff
@@ -58,5 +58,17 @@ function render() {
   for (let n = rowsNumber - 1; n > 0; n--) {
     table.deleteRow(n);
   }
+// Insert updated rows and cells
+let length = myLibrary.length;
+for (let i = 0; i < length; i++) {
+  let row = table.insertRow(1);
+  let titleCell = row.insertCell(0);
+  let authorCell = row.insertCell(1);
+  let pagesCell = row.insertCell(2);
+  let wasReadCell = row.insertCell(3);
+  let deleteCell = row.insertCell(4);
 
+  titleCell.innerHTML = myLibrary[i].title;
+  authorCell.innerHTML = myLibrary[i].author;
+  pagesCell.innerHTML = myLibrary[i].pages;
```

Comment:
I'm pretty sure your code wasn't working when you made this commit, because there are a whole bunch of missing } at the end (which show up in the next commit).  It's generally considered prudent to have committed code in a reasonable state of "cleanness" as others may check it out and try to use it.  Less of an issue on a branch that only you are using, but something to bear in mind